File: exercicios_C/1.exercSequenciais/exerc16.c
#include <stdio.h>
#include <math.h>

void main(){
    int latas, litros = 0;
    float preco, metros = 0;

    printf("informe o tamanho em metros: ");
    scanf("%f", &metros);
    litros = metros / 3;
    latas = ceil(litros / 18);
    printf("qtd de latas necessarias: %d latas.\n", latas);
    preco = latas * 80.00;
    printf("preco total: R$%0.2f.\n", preco);
}

File: exercpythonbrasil/1.exerc_sequencial/exerc16.py
metros = int(input("Informe a qtd em metros quadrados: "))
litros = metros / 3
if litros <= 18:
	print("1 lata, custando R$ 80")
else:
	latas = (litros // 18) + 1
	custo = latas * 80
	print(latas,"latas, custando R$",custo)	
    